### PR TITLE
Readme, documentation overhaul

### LINKED ISSUE #68: Overhaul readme
When I posted the link to PlasmaPy on Reddit many people were rather confused as to the aim of what we are trying to do here. I believe the materials are already in place on Riot, Reddit - it just needs some cutting and pasting, with a pinch of rewriting.

What we need to cover:

Why astronomy benefits so much from astropy and what that is

Why plasma physics is lagging behind in that and why that's detrimental

Why Python and not Fortran or c

What's our planned scope, what do we intend to do with this project

How open we are to suggestions and input from people, especially but definitely not limited to those in plasma research (very!)

How to discuss this, link to Riot chat

How to suggest features and why we need that to progress

Some of this could go into the vision statement, but at least some of it needs to be right up there in people's faces.

I'll take a bite at this within 48h if nobody else has started work on that before that time (I'll open up a pull request ASAP so people are aware things are in progress). 

Rewriting for clarity and suggestions for additional things to mention are much appreciated!

### PR DESCRIPTION
**Pull request by StanczakDominik**
Intends to close #68, #91. Heavily work in progress, but the work done can be seen [here](https://github.com/StanczakDominik/PlasmaPy/tree/PlasmaPy-68).

### CODE REVIEW

**Comment by StanczakDominik** [comment]
I'll just note that solving #36, #93 would help us clean up lots of valuable Readme real estate :D

**Comment by StanczakDominik** [comment]
I'm marking this as ready for review for now, but I intend to keep iterating on this one.

I know some of this is mostly in a less formal style, but it seems sort of unnatural to talk about doing all this *early* stuff (like the **temporary** fact we're not on PyPI yet and the only way to get PlasmaPy is via git clone) with a straight and serious face. I can do that if you all want to, though.

Anyone pointing out anything further to overhaul

@SolarDrew, @namurphy, @yopology - I've noticed your emails in multiple places in the code. Are you guys sure this is a good place for these? In the hypothetical future where PlasmaPy reaches billions and billions of users, having the Chosen Three Maintainers marked as Willing To Resolve Doubts Via Personal E-Mail might... you know... not scale.

**Comment by SolarDrew** [comment]
That is a valid point, and we should probably have a plasmapy email that a few people have access to. But then I feel like there should also be some contact detail for code authors. @Cadair, you probably know what's best-practice here.

File: README.md [hunk 1]
```diff
@@ -1,78 +1,34 @@
...
-* Murphy, Nicholas A, Huang, Yi-Min, & PlasmaPy Community. (2016, October). PlasmaPy: beginning a community developed Python package for plasma physics. Zenodo. http://doi.org/10.5281/zenodo.163752
-
-We recently created a guide on [contributing to PlasmaPy](https://github.com/PlasmaPy/PlasmaPy/blob/master/CONTRIBUTING.md), which also contains instructions on how to join our email list.  New contributors are very welcome!  
-
-If you have any questions, please contact Nick Murphy at <namurphy@cfa.harvard.edu>, Yi-Min Huang at <yiminh@princeton.edu>, or Drew Leonard at <andy.j.leonard@gmail.com>.
+[![Build Status](https://travis-ci.org/PlasmaPy/PlasmaPy.svg?branch=master)](https://travis-ci.org/PlasmaPy/PlasmaPy)
+[![Coverage Status](https://coveralls.io/repos/github/PlasmaPy/PlasmaPy/badge.svg?branch=master)](https://coveralls.io/github/PlasmaPy/PlasmaPy?branch=master)
+
+A community developed Python 3.6+ package for plasma physics in 
+the early stages. PlasmaPy intends to be for plasmas what
```
Comment (namurphy) [inline README.md]: Minor suggestion here:
```
An open source Python 3.6+ package for plasma physics in the early stages of development.
```

File: README.md [hunk 2]
```diff
@@ -1,78 +1,34 @@
...
+[Astropy](https://github.com/astropy/astropy) is for astronomy - a 
+collection of functionality commonly used and shared between plasma physicists 
+and researchers globally, running within and leveraging the open source 
+scientific Python ecosystem. The goals of this project are better described in our
```
Comment (namurphy) [inline README.md]: Instead of "better", could we say "more thoroughly"?

**Comment by namurphy** [comment]
And agreed - we should set up a PlasmaPy email list, or whatever the best practices are for that.

File: README.md (hunk 2 above)
Comment (StanczakDominik) [reply README.md]: Done!

**Pull request merged**